pkg/client/unversioned/daemon_sets_test.go

```diff
@@ -14,7 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package unversioned
+package unversioned_test
```

Comment:
Just to emphasize @xiang90's point: https://golang.org/doc/effective_go.html#package-names: "there should be no need for underscores or mixedCaps". though this rule is not strictly followed in our codebase.

"_test" is a special case. And yes, we need to use it to break cyclic imports.